metadata-ingestion/src/datahub/ingestion/source_config/operation_config.py

```diff
@@ -65,21 +65,21 @@ def validate_profile_date_of_month(cls, v: Optional[int]) -> Optional[int]:
 def is_profiling_enabled(operation_config: OperationConfig) -> bool:
     if operation_config.lower_freq_profile_enabled is False:
         return True
-    logger.info("Lower freq profiling setting is enabled.")
+    logger.debug("Lower freq profiling setting is enabled.")
```

Comment:
once the cache is in place, we can probably leave these log lines at the `info` level